app/controllers/api/v1/profiles_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Api::V1
+  class ProfilesController < ApiController
+
+    def index
+      if params[:uids].present?
+        profiles = Profile.where(uid: params[:uids])
+      else
+        profiles = Profile.all
```

Comment:
Yes your right @novotnyjakub and @robmckinnon.
Currently we don't have a story about loading all Profiles (only all Profiles in particular contexts like all Profiles or an Organisation or all Profiles of a particular type which I mentioned above)